src/main/java/uk/gov/hmcts/reform/cwrdapi/service/ValidationService.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.hmcts.reform.cwrdapi.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.cwrdapi.client.domain.CasWorkerDomain;
+import uk.gov.hmcts.reform.cwrdapi.domain.ExceptionCaseWorker;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+import javax.validation.ConstraintViolation;
+
+@Component
+public class ValidationService {
+
+    @Autowired
+    JsrValidatorInitializer<CasWorkerDomain> jsrValidatorInitializer;
+
+    @Autowired
+    AuditService auditService;
+
+    /**
+     * Returns invalid record list and JSR Constraint violations pair.
+     *
+     * @param caseWorkerProfileList List
+     * @return CasWorkerDomain list
+     */
+    public List<CasWorkerDomain> getInvalidRecords(List<CasWorkerDomain> caseWorkerProfileList) {
+        //Gets Invalid records
+        List<CasWorkerDomain> invalidRecords = jsrValidatorInitializer.getInvalidJsrRecords(caseWorkerProfileList);
```

Comment:
you can just inline it by `return jsrValidatorInitializer.getInvalidJsrRecords(caseWorkerProfileList);` instead of assigning to a variable and then returning it